mantle/platform/qemu.go

```diff
@@ -771,6 +771,10 @@ func (builder *QemuBuilder) addDiskImpl(disk *Disk, primary bool) error {
 				pId, disk.attachEndPoint))
 		}
 	} else {
+		// In the non-multipath case we can just unlink the disk now
```

Comment:
Should we make this conditional on `disk.NbdDisk` too? I can see in `prepare()` the filename is passed to `qemu-nbd`, but it's not clear whether it's safe to delete the file right after.

Reply:
Yep, good catch.  That said, at this point we should just stick all this stuff in a single temporary directory.